ql/src/java/org/apache/hadoop/hive/ql/exec/MapOperator.java

```diff
@@ -407,7 +407,7 @@ public void initEmptyInputChildren(List<Operator<?>> children, Configuration hco
       StructObjectInspector soi = null;
       PartitionDesc partDesc = conf.getAliasToPartnInfo().get(tsOp.getConf().getAlias());
       Configuration newConf = tableNameToConf.get(partDesc.getTableDesc().getTableName());
-      Deserializer serde = partDesc.getTableDesc().getDeserializer();
+      Deserializer serde = partDesc.getTableDesc().getSerDe();
```

Comment:
Nit: AbstractSerDe serDe for consistency.

Reply:
Done.